understanding/pronunciation.html

```diff
@@ -55,154 +53,116 @@
-                  the proper pronunciation would be required. Additionally, in some languages certain
-                  characters can be pronounced in different ways. In Japanese, for example, there are
-                  characters like Han characters(Kanji) that have multiple pronunciations. Screen readers
-                  may speak the characters incorrectly without the information on pronunciation. When
-                  read incorrectly, the content will not make sense to users.
+                  	たとえば、英語の同形異音異義語の "desert" (見捨てる) と"desert" (砂漠) のように、スペルは同じでも発音と意味が異なる単語がある。適切な発音を文の文脈から決定できる場合は、何も必要とされない。それができない場合は、適切な発音を決定するための何らかの仕組みが必要になる。さらに、一部の言語は、特定の文字が異なる方法で発音されることがある。例えば、日本語では、複数の発音を持つ漢字のような文字がある。スクリーンリーダーは、発音に関する情報がないと、文字を間違って話すことがある。間違って読むと、コンテンツは利用者にとって意味をなさなくなるだろう。
                   
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
-               <p>This Success Criterion may help people who: 
+               <h2>利点</h2>
+               <p>この達成基準は、次のような利用者にメリットがある:
                   
                   
                </p>
                <ul>
                   
                   
-                  <li>have difficulty decoding words</li>
+                  <li>単語を復号化しづらい。</li>
                   
                   
-                  <li>have difficulty using context to aid understanding</li>
+                  <li>前後の文脈を用いて理解するのが困難である。</li>
                   
                   
-                  <li>use technologies that read the words aloud</li>
+                  <li>音声で読み上げる支援技術を使用している。</li>
                   
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <dl>
                   
-                  <dt>Giving the reading of a person's name</dt>
+                  <dt>人名の読み方を提供</dt>
                   
-                  <dd>Web content in Japanese provides kana (Japanese phonetic syllabary characters) written
-                     next to Han characters (Kanji)  show the pronunciation of a person's name. The kana
-                     is provided to users in parentheses right after the word. Giving the reading of the
-                     words written in Han characters (Kanji) allows both sighted users and screen readers
-                     to read/pronounce the words correctly. Note that screen readers will speak the word
-                     twice: the Han characters (Kanji) that can be pronounced in a wrong way are read first
-                     and then kana is spoken in order to provide the correct reading.
+                  <dd>日本語のウェブコンテンツは、人名の発音を表している、漢字の後に書かれた仮名を提供する。仮名は、単語の直後にかっこで括弧書きで提供される。漢字で書かれた単語を読むことで、目の見える利用者とスクリーンリーダーの両方が、その語句を正しく読み/発音することができる。ただし、スクリーンリーダーは、単語を 2 回読むことになる。誤った発音が可能な漢字を先に読み、正しい読みを提供するために仮名が読み上げられる。
```

Comment:
「かっこで括弧書きで」は重複なのでどちらかで良いかなと

```suggestion
                  <dd>日本語のウェブコンテンツは、人名の発音を表している、漢字の後に書かれた仮名を提供する。仮名は、単語の直後に括弧書きで提供される。漢字で書かれた単語を読むことで、目の見える利用者とスクリーンリーダーの両方が、その語句を正しく読み/発音することができる。ただし、スクリーンリーダーは、単語を 2 回読むことになる。誤った発音が可能な漢字を先に読み、正しい読みを提供するために仮名が読み上げられる。
```